cms/bundles/forms.py

```diff
@@ -25,6 +36,8 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         - Hides the "Released" status choice as that happens on publish
         - disabled/hide the approved at/by fields
         """
+        self.dataset_api_access_token = kwargs.pop("access_token", None)
```

Comment:
Can we clarify this is the user's access token, also the token isn't specific to `dataset_api` specific (plus it's bundles api 🙈 ).

Reply:
be9e131